src/message-box.scss

```diff
@@ -0,0 +1,126 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./functions";
+@import "./dialog-placeholders";
+
+$block: #{$fd-namespace}-message-box;
+$bar: #{$fd-namespace}-bar;
+$button: #{$fd-namespace}-button;
+
+$fd-message-box-icon-text-spacing: 0.5rem;
+$fd-message-box-title-icon-text-size: 1rem;
+$fd-message-box-footer-button-min-width: 4rem;
+$fd-message-box-content-padding-x: (s: 1rem, m: 2rem, l: 2rem, xl: 3rem);
+$defaultBoxShadow: var(--sapContent_HeaderShadow);
+$errorBoxShadow: inset 0 -0.0625rem var(--sapErrorBorderColor), var(--sapContent_HeaderShadow);
+$successBoxShadow: inset 0 -0.0625rem var(--sapSuccessBorderColor), var(--sapContent_HeaderShadow);
+$warningBoxShadow: inset 0 -0.0625rem var(--sapWarningBorderColor), var(--sapContent_HeaderShadow);
+$informationBoxShadow: inset 0 -0.0625rem var(--sapInformationBorderColor), var(--sapContent_HeaderShadow);
+
+$message-box-states: (
+  "confirmation": ("iconColor": var(--sapNeutralElementColor), "icon": "\e1c4", "boxShadow": var(--sapContent_HeaderShadow)),
+  "error": ("iconColor": var(--sapNegativeElementColor), "icon": "\e0b1", "boxShadow": $errorBoxShadow),
+  "success": ("iconColor": var(--sapPositiveElementColor), "icon": "\e203", "boxShadow": $successBoxShadow),
+  "warning": ("iconColor": var(--sapCriticalElementColor), "icon": "\e201", "boxShadow": $warningBoxShadow),
+  "information": ("iconColor": var(--sapInformativeElementColor), "icon": "\e05c", "boxShadow": $informationBoxShadow)
+);
+
+.#{$block} {
+  @extend %dialog;
+
+  display: none;
+
+  // ELEMENTS
+  &__content {
+    @extend %dialog-content;
+
+    overflow: hidden;
```

Comment:
This is going to prevent popovers from overflowing Message Box window. Dialog (with changes brought in this PR) doesn't prevent `__content` overflow.